transitgateway.tf

```diff
@@ -44,13 +44,13 @@ resource "aws_ram_resource_association" "tgw" {
 }
 
 #
-# Share the resource with the other accounts that are allowed to
-# access it (currently the env*, PCA, and User Services accounts).
+# Share the resource with the other accounts that are allowed to access
+# it (currently the Domain Manger, env*, PCA, and User Services accounts).
 #
 resource "aws_ram_principal_association" "tgw" {
   provider = aws.sharedservicesprovisionaccount
 
-  for_each = merge(local.env_accounts_same_type, local.pca_account_same_type, local.userservices_account_same_type)
+  for_each = merge(local.domainmanager_account_same_type, local.env_accounts_same_type, local.pca_account_same_type, local.userservices_account_same_type)
```

Comment:
I was going to suggest that we create a `local` variable for this `merge()`, but then I realized that `local.userservices_account_same_type` isn't listed in the two `merge()`s below this.  Can you remind me why not?

Reply:
The answer you seek lies in this code comment: https://github.com/cisagov/cool-sharedservices-networking/blob/b76bd3a963e1a140dbbada41b547d661c418d88d/transitgateway.tf#L64-L66

It came about from [this PR comment thread](https://github.com/cisagov/cool-sharedservices-networking/pull/37#pullrequestreview-580719757).

Reply:
Of course I know him.  He's me!

Thanks!